components/raftstore/src/store/peer.rs

```diff
@@ -4536,6 +4539,18 @@ where
         });
         !has_overlapped_reads && !has_overlapped_writes
     }
+
+    pub fn adjust_raft_max_inflight_msgs_if_cfg_changed<T>(
```

Comment:
Better to name it `adjust_cfg_if_changed`, then we can add more config changes here in the future.

Reply:
Fixed, thanks!